app/src/main/res/layout/activity_main.xml

```diff
@@ -12,11 +12,13 @@
         android:layout_height="wrap_content"
         android:layout_marginStart="0dp"
         android:layout_marginEnd="0dp"
-        android:background="?android:attr/windowBackground"
+        android:background="@color/colorPrimary"
```

Comment:
With material theme, you can revert the changes to this file and get a white action bar with green highlighted icons and text